lib/puppet/file_serving/base.rb

```diff
@@ -46,7 +46,8 @@ def initialize(path, options = {})
   def links=(value)
     value = value.to_sym
     value = :manage if value == :ignore
-    raise(ArgumentError, ":links can only be set to :manage or :follow") unless [:manage, :follow].include?(value)
+    #TRANSLATORS ':link', ':manage', ':follow' should not be translated
+    raise(ArgumentError, _(":links can only be set to :manage or :follow")) unless [:manage, :follow].include?(value)
```

Comment:
Should we take this chance to update the message to include :ignore, since that would be accepted by the code?

Reply:
Looking at the file resource type documentation, I'm inclined not to as it doesn't list ignore at all.

links
How to handle links during file actions. During file copying, follow will copy the target file instead of the link and manage will copy the link itself. When not copying, manage will manage the link, and follow will manage the file to which the link points.

Valid values are follow, manage.

I wonder if that's something internal to the code with happens with the ignore parameter

Reply:
OK, thanks for looking into it.  Looks good otherwise.

Reply:
I looked into the git history and it appears `:ignore` has been something that would've been accepted by the code as long as the other values, so I agree it was left out of this message intentionally. 👍